src/backend/app/db/postgis_utils.py

```diff
@@ -90,10 +90,15 @@ def get_centroid(
 
 
 def geojson_to_geometry(
-    geojson: Union[FeatCol, Feature, Polygon],
+    geojson: Union[dict, FeatCol, Feature, Polygon],
```

Comment:
Could we not update to this?
```python
    outline_geojson: Union[FeatureCollection, Feature, MultiPolygon, Polygon]
```

Then handle the multipolygon geom type if passed

Reply:
since we convert multi polygon to polygon , why do we need to handle there i am a bit confused. Or do you mean to handle it there instead of the separate function `multipolygon_to_polygon`?

Reply:
Yes there is an issue when splitting MultiPolygon using splitting algorithm and need handling in Fmtm splitter and osm-rawdata. But this PR is made for the different issue. When the User uploads multipolygon and selects choose area as tasks, then we get problem while saving to database as our field outline in Project only supports Polygon. So , we need to convert the aoi with MultiPolygon type to Polygon. or need changes in order to accept MultiPolygon task or project aoi outline.
or am i missing something?